commands/mute.js

```diff
@@ -0,0 +1,60 @@
+const {modRole} = require('../config.json');
+
+module.exports = {
+	name: 'mute',
+	description: 'Muting users',
+	guildOnly: true,
+	execute(message, args) {
+        var fs = require('fs');
+
+
+        if (message.member.roles.has(modRole)){
+
+            const mutedJSONString = fs.readFileSync('muted.json','utf8');
+            console.log("String: "+mutedJSONString);
+
+            var mutedJSON = JSON.parse(mutedJSONString);
+            console.log("Keys = ",Object.keys(mutedJSON[0]));
+
+
+            args[0] = args[0].replace('<','');
+            args[0] = args[0].replace('>','');
+            args[0] = args[0].replace('@','');
```

Comment:
There must be a cleaner way of doing this, if not we should make a function to add some characters to a string before and after it, would be useful for things like mentioning users and such

Reply:
Try `message.mentions.user` or `message.mentions.members.first()`